src/lib/pipeline/FetchUnit.cc

```diff
@@ -61,6 +61,7 @@ void FetchUnit::tick() {
   uint8_t bufferOffset;
 
   // Check if more instruction data is required
+  // TODO Unsure of whether this should be minimum instruction size instead
   if (bufferedBytes_ < isa_.getMaxInstructionSize()) {
```

Comment:
I have seen somewhere that compressed instructions can be flagged during a predecode that happens when moving data into the I-cache so that this information _could_ be known during fetch. RISC-V compressed instructions are determined by the 2 least significant bits of the instruction not being 11 (i.e. 00,01,10) but I don't think that alone can determine whether it is 16 bits or the second half of a 32 bit instruction.

Reply:
Implementation changed so now irrelevant. Resolving